### PR TITLE
Ecto 3

### PR DESCRIPTION
**Pull request by mtarnovan**
Update to Ecto 3, replace Poison with Jason, upgrade Comeonin and Bcrypt, assorted fixes.

### CODE REVIEW

File: test/support/rollbackable.ex [hunk 1]
```diff
@@ -15,7 +15,8 @@ defmodule Authable.Rollbackable do
   setup do
     # Wrap this case in a transaction
     Ecto.Adapters.SQL.Sandbox.mode(@repo, :manual)
-    :ok = Ecto.Adapters.SQL.Sandbox.checkout(@repo)
+    # :ok = Ecto.Adapters.SQL.Sandbox.checkout(@repo)
```
Comment (manukall) [inline test/support/rollbackable.ex]: what's going on here?

Comment (mtarnovan) [reply test/support/rollbackable.ex]: There's a comment on the commit. Wasn't sure why a conn was already checked out so I didn't remove this line.

Comment (manukall) [reply test/support/rollbackable.ex]: hm, let's delete the commented out code then. after that it's good to merge.

**Review by manukall — APPROVED** [review]
i've deleted the commented out code from the web interface. good to merge now.

**Pull request merged**